internal/controller/monitor.go

```diff
@@ -0,0 +1,80 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	bs "github.com/redhat-developer/rhdh-operator/api/v1alpha4"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+func (r *BackstageReconciler) applyServiceMonitor(ctx context.Context, backstage *bs.Backstage) error {
+	lg := log.FromContext(ctx).WithValues("Backstage", backstage.Name)
```

Comment:
```suggestion
	lg := log.FromContext(ctx)
```
I dont think we actually need the `.WithValues("Backstage", backstage.Name)` because context is already clear, meaning it's the same reconcile loop context where the backstage instance is known